.github/actions/depot_tools/action.yaml

```diff
@@ -8,11 +8,13 @@ runs:
   using: "composite"
   steps:
       - name: Get depot tools
-        env:
-          DEPOT_TOOLS_REVISION: 22e5a04e5975a4308c15b45b34b1b120bd0c7224
+        run: git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+        shell: bash
+      - name: Bootstrap bundled python3
         run: |
-          git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
-          git -C $GITHUB_WORKSPACE/depot_tools reset --hard ${DEPOT_TOOLS_REVISION}
+          # Must be run manually as $DEPOT_TOOLS_UPDATE is 0 in CI, skipping the bootstrap.
+          source depot_tools/bootstrap_python3
+          bootstrap_python3
```

Comment:
Since this function is defined by the line above
join them (with `&&` maybe)?

Reply:
Done